tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 skipsdist = true
 isolated_build = true
-envlist = clean, lint, without-pyodbc
+envlist = clean, lint, without-extras
```

Comment:
Minor remark only, I would say without-pyodbc-and-pymongo is better, since we aren't sure what extra packages we might use in the future, I'll leave the judgement call to you

Reply:
```suggestion
envlist = clean, lint, without-pyodbc-and-pymongo
```